examples/seismic/acoustic/operators.py

```diff
@@ -25,14 +34,26 @@ def laplacian(field, m, s, kernel):
 def iso_stencil(field, m, s, damp, kernel, **kwargs):
     """
     Stencil for the acoustic isotropic wave-equation:
-    u.dt2 - H + damp*u.dt = 0
-    :param field: Symbolic TimeFunction object, solution to be computed
-    :param m: square slowness
-    :param s: symbol for the time-step
-    :param damp: ABC dampening field (Function)
-    :param kwargs: forwad/backward wave equation (sign of u.dt will change accordingly
-    as well as the updated time-step (u.forwad or u.backward)
-    :return: Stencil for the wave-equation
+    u.dt2 - H + damp*u.dt = 0.
+
+    Parameters
+    ----------
+    field : TimeFunction
+        The computed solution.
+    m : float
+        Square slowness.
+    s :
+        Time-step.
+    damp : `Function`
+        ABC dampening field.
+    kwargs : dict
+        Forward/backward wave equation (sign of u.dt will change accordingly
+    as well as the updated time-step (u.forwad or u.backward).
+
+    Returns
```

Comment:
We can probably drop this as it is already clear what will this function return?